articles/quickstart/spa/auth0-react/02-calling-an-api.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Call an API
+description: This tutorial demonstrates how to make API calls for protected resources on your server.
+budicon: 448
+topics:
+  - quickstarts
+  - spa
+  - react
+  - login
+github:
+  path: Sample-01
+contentType: tutorial
+useCase: quickstart
+---
+<!-- markdownlint-disable MD002 MD034 MD041 -->
+
+Your React application may need to access protected data from an API that you are also [securing with Auth0](https://auth0.com/docs/microsites/protect-api/protect-api). For that use case, you'll need to retrieve an access token to authorize any calls you make to that secured API.
+
+For this example, you'll use the [Auth0 Management API](https://auth0.com/docs/api/management/v2) to quickly test how to make calls to a protected external API.
+
+:::note
+If you followed the [previous section where you added user log in](/quickstart/spa/auth0-react#add-login-to-your-application), make sure that you log out of your application as you'll need a new access token to call APIs.
+:::
+
+For the Auth0 React SDK to configure the access token correctly, you need to pass an `audience` and a `scope` property to the `Auth0Provider` as follows:
+
+```javascript
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+import { Auth0Provider } from "@auth0/auth0-react";
+
+ReactDOM.render(
+  <Auth0Provider
+    domain="${account.namespace}"
+    clientId="${account.clientId}"
+    redirectUri={window.location.origin}
+    audience={`https:// <%= "${process.env.REACT_APP_AUTH0_DOMAIN}" %>/api/v2/`}
+    scope="read:users update:current_user_metadata"
+  >
+    <App />
+  </Auth0Provider>,
+  document.getElementById("root")
+);
+```
+
+The set up is similar to the one discussed in the [Configure the `Auth0Provider` component](/quickstart/spa/auth0-react#configure-the-auth0provider-component) section: you wrap your root component with `Auth0Provider` and use the "Settings" values of the single-page application you've registered with Auth0 to populate the values of `domain` and `clientId`.
+
+However, your React application needs to pass an access token when it calls a target API to access private user data. You [request an access token](https://auth0.com/docs/tokens/guides/get-access-tokens) in a format that the API can verify by passing the `audience` and `scope` to `Auth0Provider`.
+
+The `audience` lets the API verify that it has received an access token intended for it and not for some other API. The `scope` lets the API understand the permissions that the client application has in order to apply authorization properly.
```

Comment:
I would write this differently: 

The `scope` attribute instructs Auth0 the scopes the client needs to operate. Auth0 will decide whether or not it issues a token with those scopes. This decision is based on multiple factors ranging from authentication of the user to policies running in Rules. If it is successful, then the token will contain the scopes requested and the API will validate that token and check for those scopes.

Reply:
I am not too clear on what this mean 🤔  Asked for clarification on the Slack channel but I adapted this in the content regardless.